### PR TITLE
Show random color button

### LINKED ISSUE #531: show random color button for ownCloud 9.1+
With 5bae6a108e2fb5937539d0d45bf8569bbec57b01 I hid the random color picker.
It doesn't work with ownCloud 9. We should only display it for ownCloud 9.1+

### PR DESCRIPTION
**Pull request by georgehrke**
reintroduce random color button
fixes https://github.com/owncloud/calendar/issues/531

Needs to be tested on:
- [x] stable9
- [x] stable9.1
- [x] master

### CODE REVIEW

File: tests/js/unit/utility/colorUtilitySpec.js [hunk 1]
```diff
@@ -96,10 +96,10 @@ describe('ColorUtility shared tests', function () {
 
 	it('should call forward the call to hslToRgb', function () {
 		expect(ColorUtility._hslToRgb(6, 70, 68));
-		expect(hslToRgb).toHaveBeenCalledWith(6, 70, 68);
+		expect(hslToRgb).toHaveBeenCalledWith(6, .7, .68);
```
Comment (tcitworld) [inline tests/js/unit/utility/colorUtilitySpec.js]: JSHint shows warnings and the grunt tasks fails, so you should add a leading 0 I guess.

> A leading decimal point can be confused with a dot

**Comment by tcitworld** [comment]
When creating a new calendar, the random color is not shown on the same line (french screenshot, but the same with en)
random_color

**Comment by georgehrke** [comment]
@raghunayyar Please take a look at this new line issue mentioned by @tcitworld

**Comment by raghunayyar** [comment]
Sure, will have a look at this asap.

**Comment by raimund-schluessler** [comment]
@raghunayyar You can have a look at the Tasks app. There it is aligned correctly.

**Comment by georgehrke** [comment]
@raimund-schluessler He already pushed a commit that aims to fix it :)
Haven't tested it yet.

**Comment by georgehrke** [comment]
agenda - nextcloud chromium today at 11 44 33 am
Aligning issue seems to be fixed :)

I'll now test this on the other branches

**Comment by georgehrke** [comment]
Tested on all supported server versions 👍

**Comment by skjnldsv** [comment]
Well done!! :+1:

**Comment by EtlamGit** [comment]
No, not well done!
The line with `flex-wrap` was intentionally placed there. If missing it destroys the layout when custom colors are defined. See the PullRequest #602 as reference.

The problem shown here looks like "too many" colors are defined to fit into one row. Therefore it wraps to the next row (so an intended behavior). A "good" fix would do one of these things:
- reduce the number of colors
- make the colors boxes a little bit smaller
- reduce the left and right margin
- widen the complete navigation pane
- remove the random button

**Pull request merged**